### PR TITLE
add sidebar plugin guide

### PR DESCRIPTION
**Pull request by lx0f**

### CODE REVIEW

File: advanced/sidebar.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# Sidebar Plugin
```
Comment (Ducksss) [inline advanced/sidebar.md:1]: Change this to "Using the Sidebar Plugin"

File: _sidebar.md [hunk 2]
```diff
@@ -6,6 +6,7 @@
   * [Docsify configuration](advanced/docsify)
   * [Markdown features](advanced/markdown-features)
   * [Using and Customising the Accordion Plugin](advanced/accordion)
+  * [Using the Sidebar Plugin](advanced/sidebar)
```
Comment (Ducksss) [inline _sidebar.md:9]: Change this to "Sidebar Plugin"

File: advanced/sidebar.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+The Sidebar plugin allows you to nest your page links into interactive folders
+you can collapse and uncollapse!
```
Comment (Ducksss) [inline advanced/sidebar.md:4]: is this line break intentional?

File: assets/example-sidebar.gif
Comment (Ducksss) [inline assets/example-sidebar.gif:1]: v nice gif ^_^

**Review by Ducksss — APPROVED** [review]

File: advanced/sidebar.md (hunk 3 above)
Comment (lx0f) [reply advanced/sidebar.md:4]: yez so it doesn't overflow in my editor, y leh? is it affecting the rendering 😟

Comment (lx0f) [reply advanced/sidebar.md:4]: just checked, it doesn't break affect how it looks when rendered 👍

File: assets/example-sidebar.gif
Comment (lx0f) [reply assets/example-sidebar.gif:1]: xie xie

**Pull request merged**